src/patternfly/demos/Page/examples/Page.md

```diff
@@ -11,6 +11,12 @@ wrapperTag: div
 {{> page-template page-template--id="page-demo-basic"}}
 ```
 
+### Full height page
+Using the `pf-m-full-height` modifier on the page component eliminates the need to ensure that the `<html>` and `<body>` tags, and any other tags above `.pf-c-page`, have height set to 100%.
```

Comment:
```suggestion
Using the `.pf-m-full-height` modifier on the page component eliminates the need to ensure that the `<html>` and `<body>` tags, and any other ancestors of `.pf-c-page`, have height set to 100%.
```

Reply:
Oh sorry I meant to give more context here. Wanted to say maybe use `.` with the class and/or say "modifier _class_" just so it's clear it's a class. Also that we use "ancestors" on the react side when referring to the elements above page, that seems a little clear. https://www.patternfly.org/v4/components/page/react-demos

> To make the page take up the full height, it is recommended to set the height of all ancestor elements up to the page component to 100%